shell/platform/common/accessibility_bridge.cc

```diff
@@ -382,7 +382,7 @@ void AccessibilityBridge::SetIntAttributesFromFlutterUpdate(
         static_cast<int32_t>(
             flags & FlutterSemanticsFlag::kFlutterSemanticsFlagIsMixedCheck
                 ? ax::mojom::CheckedState::kMixed
-                : flags & FlutterSemanticsFlag::kFlutterSemanticsFlagIsChecked
```

Comment:
Watch the formatting here.

Reply:
Most recently this is formatted according to the output of the linux_unopt test.